core/src/main/scala/kafka/server/ApiVersionManager.scala

```diff
@@ -86,14 +100,7 @@ class DefaultApiVersionManager(
         finalizedFeatures.features.map(kv => (kv._1, kv._2.asInstanceOf[java.lang.Short])).asJava,
         finalizedFeatures.epoch,
         controllerApiVersions.orNull,
-        listenerType)
-  }
-
-  override def enabledApis: collection.Set[ApiKeys] = {
```

Comment:
Do we no longer need these methods? Or are they just handled elsewhere? (Ie, ApiKeys)